protos/graph.proto

```diff
@@ -12,13 +12,18 @@ message Put {
   uint64 objref = 1; // The objref for the object that was put
 }
 
+message Get {
+  uint64 objref = 1; // The objref for the object that is retrieved
+}
+
 // This is used internally by the scheduler. From the scheduler's perspective,
 // the submission of tasks (via SubmitTask) and the submission of puts (via
 // PutObj) look very similar, and so it is useful to be able to handle them
 // together (for example in the computation graph).
 message Operation {
   Task task = 1;
   Put put = 2;
+  Get get = 4;
```

Comment:
You are right that it is not required for fault tolerance, but it is good for visualizing, to show people where potentially expensive transfers are happening